hook/hooktest/serviceeventkind_string.go

```diff
@@ -0,0 +1,17 @@
+// generated by stringer -type=ServiceEventKind; DO NOT EDIT
+
+package hooktest
+
+import "fmt"
+
+const _ServiceEventKind_name = "ServiceEventInstallServiceEventStartServiceEventErrorServiceEventStopServiceEventRemove"
+
+var _ServiceEventKind_index = [...]uint8{0, 19, 36, 53, 69, 87}
+
+func (i ServiceEventKind) String() string {
```

Comment:
this function feels like black magic :) i'd prefer something simpler like defining a variable containing []string values of all ServiceEventKinds instead of shoving it all in on string..